### PR TITLE
SF| collect test logs after test finish

### PR DESCRIPTION
**Pull request by Avilir**
In this PR i fix the logs collection to run in any case  (success/failed) and not only in case of success run, so we can investigate benchmark failure.

### CODE REVIEW

File: tests/e2e/performance/io_workload/test_small_file_workload.py [hunk 1]
```diff
@@ -500,13 +500,36 @@ def generate_kibana_link(self, index, columns):
...
+        for clpod in self.full_client_list:
+            test_logs = self.pod_obj.exec_oc_cmd(f"logs {clpod}", out_yaml_format=False)
+            log_file_name = f"{self.full_log_path}/{clpod}-pod.log"
+            try:
+                with open(log_file_name, "w") as f:
+                    f.write(test_logs)
+                log.info(f"The Test log can be found at : {log_file_name}")
+            except Exception:
+                log.warning(f"Cannot write the log to the file {log_file_name}")
+        log.info("Logs from all client pods got successfully")
```
Comment (pintojoy) [inline tests/e2e/performance/io_workload/test_small_file_workload.py:523]: Can we rephrase this to "Logs of all the client pods have been collected successfully"

**Review by pintojoy — APPROVED** [review]
LGTM

**Review by keesturam — APPROVED** [review]
LGTM too.
